README.md

```diff
@@ -174,18 +194,18 @@ The HtmldevBundle loads the example/dummy data for the components from Yaml file
 For example, it loads a `htmldev/data/buttons.yml`:
 
 ```yaml
--
-    styleguide_title: Blue button
-    styleguide_type: buttons/text
-    text: Button
-    color: blue
--
-    styleguide_title: White button outline
-    styleguide_type: buttons/text
-    styleguide_dark: true
-    outline: true
-    text: Button
-    color: white
+    -
+        styleguide_title: Blue button
+        styleguide_type: buttons/text
+        text: Button
+        color: blue
+    -
+        styleguide_title: White button outline
+        styleguide_type: buttons/text
+        styleguide_dark: true
+        outline: true
+        text: Button
+        color: white
```

Comment:
Is dit nodig? Ik weet dat PhpStorm het niet goed interpreteert, maar dat is de fout van PhpStorm. De Markdown syntax was hier wel gewoon correct.

Github geeft dit ook goed weer, zie: https://github.com/zicht/htmldev-bundle

Reply:
Dat wist ik niet he, of het de fout was van PHPStorm of niet 😋  Ik heb in dat doc zitten editten dus ik dacht dat het misschien door mijn toedoen verschoven was :)